### PR TITLE
allow NLU-only model in /status endpoint

### PR DESCRIPTION
**Pull request by ricwo**
**Proposed changes**:
- allow NLU-only model in /status endpoint

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: rasa/server.py [hunk 1]
```diff
@@ -66,13 +66,17 @@ def _docs(sub_url: Text) -> Text:
     return DOCS_BASE_URL + sub_url
 
 
-def ensure_loaded_agent(app: Sanic):
-    """Wraps a request handler ensuring there is a loaded and usable agent."""
+def ensure_loaded_agent(app: Sanic, allow_nlu_only: bool = False):
+    """Wraps a request handler ensuring there is a loaded and usable agent.
+
+    If `allow_nlu_only is `True`, consider the agent ready event if no policy
```
Comment (ricwo) [inline rasa/server.py]:
```suggestion
    If `allow_nlu_only` is `True`, consider the agent ready event if no policy
```

**Comment by tmbo** [comment]
@ricwo I modified this a bit, avoiding the boolean (I think we should avoid boolean flags for functions in most cases) - can you take a look at the change and check if this makes sense?

**Comment by ricwo** [comment]
👍 looks good

**Comment by tmbo** [comment]
@ricwo can you take this PR over again? (travis seems to fail on status tests)

**Pull request merged**